common/src/test/java/io/pravega/common/util/SortedDequeTests.java

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
I think I missed some files that have `<p>` in the license header, here is the list I got:

SortedDequeTests.java
Write.java
WriteQueue.java
WriteQueueTests.java
QueueStats.java

could you fix those, please?